test/loki_service.cc

```diff
@@ -102,7 +102,9 @@ const std::vector<http_request_t> valhalla_requests{
 };
 
 const std::vector<std::pair<uint16_t, std::string>> valhalla_responses{
-    {200, "{}"},
+    {200,
+     R"({"has_admins":false,"actions":["'\/locate'","'\/route'","'\/height'","'\/sources_to_targets'","'\/optimized_route'","'\/isochrone'","'\/trace_route'","'\/trace_attributes'","'\/transit_available'","'\/expansion'","'\/centroid'","'\/status'"],"has_live_traffic":false,"has_timezones":false,"has_tiles":false,"bbox":"{\"features\":[],\"type\":\"FeatureCollection\"}","version":")" VALHALLA_VERSION
+     R"("})"},
```

Comment:
should we take those out entirely and have a separate test file for the status action?